cmd/create/cluster/runner.go

```diff
@@ -69,12 +102,29 @@ func (r *runner) run(ctx context.Context, _ *cobra.Command, _ []string) error {
 		}
 	}
 
+	var organization string
+	{
+		options := v1.ListOptions{
+			LabelSelector: fmt.Sprintf("%s=%s", "giantswarm.io/conformance-testing", "true"),
+		}
+		organizations, err := k8sClient.G8sClient().SecurityV1alpha1().Organizations().List(ctx, options)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+		if len(organizations.Items) == 0 {
+			return microerror.Mask(notAvailableOrganizationError)
+		}
+
+		rand.Seed(time.Now().Unix())
```

Comment:
I'd propose this to be moved to `func init()` so that it can be also overridden in tests if determinism is needed.